cylc/rose/entry_points.py

```diff
@@ -71,8 +71,7 @@ def post_install(srcdir=None, opts=None, rundir=None):
         srcdir=srcdir, opts=opts, rundir=rundir
     )
     # Finally dump a log of the rose-conf in its final state.
-    if results['fileinstall']:
```

Comment:
We set `results['fileinstall']` at line 70 - this condition will never be false.